### DISCUSSION TITLE
How to add SQL triggers to the DB using Prisma Migrate?

### DISCUSSION THREAD

**Discussion by nikolasburk** [prisma/prisma#5861] [Q&A]
How can I add SQL triggers to the DB using Prisma Migrate? 

For example:

```sql
CREATE FUNCTION quote_tsvector_trigger() RETURNS trigger AS $$
begin
  new.textsearchable_quote_index_col :=
  setweight(to_tsvector('english', coalesce(new.content, '')), 'A')
  || setweight(to_tsvector('english', coalesce(new.author, '')), 'B');
  return new;
end
$$ LANGUAGE plpgsql;
CREATE TRIGGER tsvectorupdate BEFORE INSERT OR UPDATE
    ON "Quotes" FOR EACH ROW EXECUTE PROCEDURE quote_tsvector_trigger();
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
